### PR TITLE
removes revolver snubnose, makes revolvers fit in bags again

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Revolvers are bad
Backpack man meme got nerfed to the ground to non existance anyway, and if it's such an issue i'll just add a storage limit of revolvers or make them bulky and have a bypass
Snubs are ass and they're a terrible idea to keep a meme that's already dead anyway
With slowdown, 0.15 fire delay and decent damage at the cost of mag capacity and sustainablity they'll be decent sidearms atleast
Deletes tp-44 snub as we no longer need it

## Why It's Good For The Game
Revolvers are awful and snubnose did nothing to solve the issues it created, you can't vend out 15 bags full of revolvers anymore anyway because bag grabbing got nerfed to the ground and the screenshake is gone, aswell as the stagger

## Changelog
:cl:
balance: tp-44 revolver snubnoses are gone, tp-44 no longer staggers at all, revolver now fits normally everywhere again, knockback is gone
/:cl:

### CODE REVIEW

**Comment by Ieteihowmee** [comment]
Good.

**Comment by Alexeivichi** [comment]
Xenos on suicide watch again. Time to go full gamer.

**Comment by SplinterGP** [comment]
> Revolvers are awful and snubnose did nothing to solve the issues it created, you can't vend out 15 bags full of revolvers anymore anyway because bag grabbing got nerfed to the ground and the screenshake is gone, aswell as the stagger

bag grabbing never got nerfed, indeed you can still take it out and take stuff instantly, it just has been added so you can pull it in your back without taking it out by having a delay instead of not being able to, and you can still use satchels.

also i don't know why you cant put the stagger and stuff removal just on snubnose, this is overall a nerf to who doenst use revolver by putting them on bags and shit.

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -355,10 +355,7 @@ datum/ammo/bullet/revolver/tp44
 	damage = 40
 
 /datum/ammo/bullet/revolver/tp44/on_hit_mob(mob/M,obj/projectile/P)
-	if(SEND_SIGNAL(P.shot_from, COMSIG_REVOLVER_AMMO_HIT_MOB) & COMSIG_REVOLVER_AMMO_SNUBNOSE_BARREL)
-		staggerstun(M, P, stagger = 1, slowdown = 0.5, knockback = 1, shake = 0)
```
Comment (TiviPlus) [inline code/modules/projectiles/ammo_datums.dm:359]: You didnt delete these signals

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by InterroLouis** [comment]
Don't forget the sprite changes from the snubnose addition need to be undone.

**Comment by SilencedMP5A5** [comment]
now you can add your funny magnum revolver, mjp 💯

**Comment by Losenis** [comment]
Revolverspam is bad because screenshake + stagger on full auto tier firerate that made any escape abilities effectively useless while the reload was so ass it was better to pack multiple of them instead of actually reloading the fucking thing, who the FUCK THOUGHT FUCKING OVER BACKPACKS AND REVOLVER STORAGE WAS THE SOLUTION

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**